python/lsst/meas/base/apertureFlux.cc

```diff
@@ -0,0 +1,116 @@
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the LSST License Statement and 
+ * the GNU General Public License along with this program.  If not, 
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "lsst/pex/config/pybind11.h"
+#include "lsst/meas/base/pybind11.h"
+
+#include "lsst/meas/base/ApertureFlux.h"
+
+namespace py = pybind11;
+using namespace pybind11::literals;
+
+namespace lsst {
+namespace meas {
+namespace base {
+
+namespace {
+    template <typename T, class PyClass>
+    void declareComputeFluxes(PyClass & cls) {
+        namespace afwImage = lsst::afw::image;
+        namespace afwEllipses = lsst::afw::geom::ellipses;
+        using Control = ApertureFluxAlgorithm::Control;
+        using Result = ApertureFluxAlgorithm::Result;
+
+        cls.def_static("computeSincFlux", (Result (*) (afwImage::Image<T> const &,
+                                                       afwEllipses::Ellipse const &,
+                                                       Control const &)) &ApertureFluxAlgorithm::computeSincFlux,
+            "image"_a, "ellipse"_a, "ctrl"_a=Control());
+        cls.def_static("computeSincFlux", (Result (*) (afwImage::MaskedImage<T> const &,
+                                                       afwEllipses::Ellipse const &,
+                                                       Control const &)) &ApertureFluxAlgorithm::computeSincFlux,
+            "image"_a, "ellipse"_a, "ctrl"_a=Control());
+        cls.def_static("computeNaiveFlux", (Result (*) (afwImage::Image<T> const &,
+                                                        afwEllipses::Ellipse const &,
+                                                        Control const &)) &ApertureFluxAlgorithm::computeNaiveFlux,
+            "image"_a, "ellipse"_a, "ctrl"_a=Control());
+        cls.def_static("computeNaiveFlux", (Result (*) (afwImage::MaskedImage<T> const &,
+                                                        afwEllipses::Ellipse const &,
+                                                        Control const &)) &ApertureFluxAlgorithm::computeNaiveFlux,
+            "image"_a, "ellipse"_a, "ctrl"_a=Control());
+        cls.def_static("computeFlux", (Result (*) (afwImage::Image<T> const &,
+                                                   afwEllipses::Ellipse const &,
+                                                   Control const &)) &ApertureFluxAlgorithm::computeFlux,
+            "image"_a, "ellipse"_a, "ctrl"_a=Control());
+        cls.def_static("computeFlux", (Result (*) (afwImage::MaskedImage<T> const &,
```

Comment:
Yes, I should have used a templated wrapper.

Can you point me to where the abbreviations are discouraged? The [style guide](https://developer.lsst.io/coding/cpp_style_guide.html#a-definition-or-abbreviated-namespace-should-be-used-when-defining-symbols) doesn't seem to think so.